README.md

```diff
@@ -4,6 +4,8 @@ This module is the main repository for both **Xamarin.iOS** and **Xamarin.Mac**.
 
 These frameworks allow us to create native iOS, tvOS, watchOS and Mac applications using the same UI controls we would in Objective-C and Xcode, except with the flexibility and elegance of a modern language (C#), the power of the .NET Base Class Library (BCL), and two first-class IDEs&mdash;Xamarin Studio and Visual Studio&mdash;at our fingertips.
 
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/xamarin/xamarin-macios?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
Good idea but it might be a good place to state this is not a support channel but a place for contributors to discuss the SDK